checkov/common/output/csv.py

```diff
@@ -47,23 +47,19 @@ def __init__(self) -> None:
         self.iac_resource_cache: set[str] = set()  # used to check, if a resource was already added
 
     def add_report(self, report: Report, git_org: str, git_repository: str) -> None:
-        if report.check_type == CheckType.SCA_PACKAGE:
+        if report.check_type in [CheckType.SCA_PACKAGE, CheckType.SCA_IMAGE]:
```

Comment:
```suggestion
        if report.check_type in (CheckType.SCA_PACKAGE, CheckType.SCA_IMAGE):
```
for a static list a tuple is always the best choice 🙂